api/server/router/image/image_routes.go

```diff
@@ -45,9 +44,13 @@ func (s *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrite
 	version := httputils.VersionFromContext(ctx)
 	if versions.GreaterThanOrEqualTo(version, "1.32") {
 		apiPlatform := r.FormValue("platform")
-		platform = system.ParsePlatform(apiPlatform)
-		if err = system.ValidatePlatform(platform); err != nil {
```

Comment:
See my other comment regarding platform compatibility

Reply:
I think it should looks like below code snippet:
···
m, err := platforms.Parse(apiPlatform)
...
err = m.Validate()
```